src/components/BorrowerProfile/BorrowerSocialShare.vue

```diff
@@ -0,0 +1,293 @@
+<template>
+	<section class="share hide-for-print">
+		<h2 class="share__title">
+			Help by sharing
+		</h2>
+		<div class="share__social social">
+			<a
+				data-testid="share-facebook-button"
+				class="social__btn social__btn--facebook"
+				:href="facebookShareUrl"
+				v-kv-track-event="['Borrower Profile', 'Social-Share-Lightbox', 'click-Facebook-share']"
+			>
+				<kv-icon name="facebook" title="Facebook" class="social__icon" />
+			</a>
+			<a
+				data-testid="share-twitter-button"
+				class="social__btn social__btn--twitter"
+				:href="twitterShareUrl"
+				target="_blank"
+				rel="noopener"
+				v-kv-track-event="['Borrower Profile', 'Social-Share-Lightbox', 'click-Twitter-share']"
+				@click="$showTipMsg('Thanks for tweeting!')"
+			>
+				<kv-icon name="twitter" title="Twitter" class="social__icon" />
+			</a>
+			<a
+				data-testid="share-email-button"
+				class="social__btn social__btn--email"
+				:href="emailShareUrl"
+				target="_blank"
+				rel="noopener"
+				v-kv-track-event="['Borrower Profile', 'Social-Share-Lightbox', 'click-Email-share']"
+				@click="$showTipMsg('Thanks for sharing!')"
+			>
+				<kv-icon name="email" title="Email" class="social__icon" />
+			</a>
+			<a
+				data-testid="share-copy-link-button"
+				class="social__btn social__btn--link"
+				:disabled="copyStatus.disabled"
+				v-kv-track-event="['Borrower Profile', 'Social-Share-Lightbox', 'click-Copy-link-share']"
+				@click="copyLink"
+			>
+				<kv-icon name="link" tile="Copy Link" class="social__icon" />
+				<span
+					class="social__tooltip tw-p-1 tw-text-small"
+					:class="copyStatus.class"
+				>
+					{{ this.copyStatus.text }}
+				</span>
+			</a>
+		</div>
+	</section>
+</template>
+
+<script>
+import _map from 'lodash/map';
+import clipboardCopy from 'clipboard-copy';
+import KvIcon from '@/components/Kv/KvIcon';
+
+export default {
+	inject: ['apollo'],
+	components: {
+		KvIcon
+	},
+	props: {
+		borrowerName: {
+			type: String,
+			default: ''
+		},
+		sector: {
+			type: String,
+			default: ''
+		}
+	},
+	data() {
+		return {
+			copyStatus: {
+				class: '',
+				disabled: false,
+				text: ''
+			},
+			shareMessage: '',
+			shareLink: `https://${this.$appConfig.host}${this.$route.path}`
+		};
+	},
+	computed: {
+		facebookShareUrl() {
+			return this.getFullUrl('https://www.facebook.com/dialog/share', {
+				app_id: this.$appConfig.fbApplicationId,
+				display: 'page',
+				href: `${this.shareLink}?utm_source=facebook.com&utm_medium=social&utm_campaign=social_share_checkout`,
+				redirect_uri: `${this.shareLink}?kiva_transaction_id=${this.$route.query.kiva_transaction_id}`,
```

Comment:
There isn't a transaction id on this page, so let's use the `href` value for `redirect_uri` as well.

Reply:
Forgot to ask about that, sorry!